cascading-hadoop3-io/src/main/java/cascading/tap/hadoop/io/MultiInputFormat.java

```diff
@@ -256,7 +255,7 @@ public RecordReader operate() throws Exception
         @Override
         public boolean rethrow( Exception exception )
           {
-          return !( exception.getCause() instanceof S3ServiceException );
+          return true;
           }
         } );
```

Comment:
TODO: figure where S3ServiceException used to come from it is no longer on the classpath

Reply:
@cwensel I might be missing something but I don't see `S3ServiceException` in the signature of the `getRecordReader` func. I narrowed the exception type to `IOException` in https://github.com/cwensel/cascading/pull/82/commits/9c0a0dfa35fe1a3af7745615b1c2d40ff3e2420c and it seems to be happy. 

so it seems to me like we should just always return false here like so: https://github.com/cwensel/cascading/pull/82/commits/aa55165ae77103971bf71cd0966557e4ae53d56b